Makefile

```diff
@@ -0,0 +1,40 @@
+SHELL := /bin/bash
+
+CURRENT_PATH = $(shell pwd)
+REPO = # ghcr.io
+IMAGE = techneatium/techs-checks-latex
+TAG = latest
+FULL_IMAGE = ${REPO}${IMAGE}:${TAG}
+TARGET = F14_CheatSheet.tex
+
+.PHONY: build_container
+prepare_image: clean_fonts
+	docker build -t ${FULL_IMAGE} .
+
+.PHONY: build_pdf
+build_pdf:
+	mkdir -p ./build
+	source scripts/set_build_info.sh
```

Comment:
This line always yields the following error:
```
source scripts/set_build_info.sh
make: source: No such file or directory
make: *** [build_pdf] Error 1
```

Reply:
Are you using a lightweight shell like `ash` or `sh`? I'm not sure from the error if it's complaining that `source` cannot be found, or the shell can't source `scripts/set_build_info.sh`.